app/models/manageiq/providers/embedded_terraform/automation_manager/openstack_credential.rb

```diff
@@ -0,0 +1,131 @@
+  COMMON_ATTRIBUTES = [
+    {
+      :component  => 'text-field',
+      :label      => N_('Username'),
+      :helperText => N_('The username to use to connect to OpenStack'),
+      :name       => 'userid',
+      :id         => 'userid',
+      :isRequired => true,
+      :validate   => [{:type => 'required'}],
+    },
+    {
+      :component  => 'password-field',
+      :label      => N_('Password (API Key)'),
+      :helperText => N_('The password or API key to use to connect to OpenStack'),
+      :name       => 'password',
+      :id         => 'password',
+      :type       => 'password',
+      :isRequired => true,
+      :validate   => [{:type => 'required'}],
+    },
+  ].freeze
+
+  EXTRA_ATTRIBUTES = [
+    {
+      :component  => 'text-field',
+      :label      => N_('Host (Authentication URL)'),
+      :helperText => N_('The host to authenticate with. For example, https://openstack.business.com/v2.0'),
+      :name       => 'host',
+      :id         => 'host',
+      :maxLength  => 1024,
+      :isRequired => true,
+      :validate   => [{:type => 'required'}],
+    },
+    {
+      :component  => 'text-field',
+      :label      => N_('Project (Tenant Name)'),
+      :helperText => N_('This is the tenant name. This value is usually the same as the username'),
+      :name       => 'project',
+      :id         => 'project',
+      :maxLength  => 100,
+      :isRequired => true,
+      :validate   => [{:type => 'required'}],
+    },
+    {
+      :component  => 'text-field',
+      :label      => N_('Domain Name'),
+      :helperText => N_('OpenStack domains define administrative boundaries. It is only needed for Keystone v3 authentication URLs'),
+      :name       => 'domain',
+      :id         => 'domain',
+      :maxLength  => 100,
+    },
+    {
+      :component  => 'text-field',
+      :label      => N_('Region Name'),
+      :helperText => N_('The region to create resources on'),
+      :name       => 'region',
+      :id         => 'region',
+      :maxLength  => 100,
+    },
+    {
+      :component      => 'password-field',
+      :label          => N_('CA certificate'),
+      :helperText     => N_('The custom CA certificate when communicating over SSL'),
+      :componentClass => 'textarea',
+      :name           => 'cacert',
+      :id             => 'cacert',
+      :type           => 'password',
+    },
+    {
+      :component      => 'password-field',
+      :label          => N_('Client Certificate'),
+      :helperText     => N_('The client certificate for SSL client authentication.'),
+      :componentClass => 'textarea',
+      :name           => 'clientcert',
+      :id             => 'clientcert',
+      :type           => 'password',
+    },
+    {
+      :component  => 'checkbox',
+      :label      => N_('Trust self-signed SSL certificates ?'),
+      :helperText => N_('Trust self-signed SSL certificates ?'),
```

Comment:
Is this the same thing as VERIFY_NONE (when insecure true) vs. VERIFY_PEER (when insecure false)?  We call that `SSL verification` or `SSL Verify Mode` elsewhere.

Reply:
Yes,  we can improve the label text ,

Reply:
Though by default `insecure` is false by default, so is OFF or 'unchecked'. 
https://registry.terraform.io/providers/terraform-provider-openstack/openstack/latest/docs#insecure

 If we use `Verify SSL`  as label, then meaning changes, ...   
 the checkbox needs to be 'ON' or 'checked' by default.  Just don't know enough DDF yet, to know how to make checkbox 'checked'  by default.

Reply:
@jrafanie ,  any other label suggestion ?

Reply:
I agree, we should use positive terms "Verify SSL" with it checked/enabled by default.  We can with @GilbertCherrie if we're not seeing it "on" by default.

Reply:
We can merge as is and I'll work with @GilbertCherrie if I can't figure out how to correctly switch the logic.  I like "verify ssl" true/false as insecure "false" is a double negative and hard to understand.